scripts/generate-gh-release-notes.py

```diff
@@ -19,23 +20,15 @@
 
 Requires Python3.6+.
 """
-import os
 import re
 import sys
 from pathlib import Path
+from typing import Sequence
 
-import github3
 import pypandoc
 
 
-def publish_github_release(slug, token, tag_name, body):
-    github = github3.login(token=token)
-    owner, repo = slug.split("/")
-    repo = github.repository(owner, repo)
-    return repo.create_release(tag_name=tag_name, body=body)
-
-
-def parse_changelog(tag_name):
+def parse_changelog(tag_name: str) -> str:
     p = Path(__file__).parent.parent / "doc/en/changelog.rst"
     changelog_lines = p.read_text(encoding="UTF-8").splitlines()
```

Comment:
@nicoddemus BTW the `title_regex` just below doesn't handle rc releases

Reply:
Good catch, will fix it.

Reply:
Fixed, thanks for the catch!